src/javafundamentals/collections/list/Ex03MaxValue.java

```diff
@@ -1,9 +1,31 @@
 package javafundamentals.collections.list;
 
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
 /**
  * Create a List with 20 random numbers.
  * Find the maximum value of the list and print it.
  */
 
 public class Ex03MaxValue {
+    public static void main(String[] args) {
+        int myLength = 20;
+        System.out.println(returnLargest(createList(myLength)));
+    }
+
+    private static List<Integer> createList(int length) {
+        List<Integer> myList = new ArrayList<>();
+        for (int i = 0; i < length; i++) {
+            myList.add((int) (Math.random() * 100));
+        }
+        System.out.println(myList);
+        return myList;
+    }
+
+    private static Integer returnLargest(List<Integer> list) {
+        return Collections.max(list);
```

Comment:
Smart! you discovered the Collections library, it has some useful methods on it.
Great rule for programmers: DRY (dont repeat yourself) so if someone did it already it is good to use it!